src/nanodbc.h

```diff
@@ -1019,6 +1019,10 @@ class connection
     /// \brief Returns the native ODBC environment handle.
     void* native_env_handle() const;
 
+
+    /// \brief Returns information from the ODBC connection as a string.
+    string_type get_string_info(short info_type) const;
```

Comment:
IMO, `get_info` would reflect correspondence to `SQLGetInfo`. The return type indicates the info is returned as a string.